homeassistant/components/fing/const.py

```diff
@@ -0,0 +1,6 @@
+"""Const for the Fing integration."""
+
+AGENT_IP = "agent_ip"
+AGENT_PORT = "agent_port"
+AGENT_KEY = "agent_key"
```

Comment:
I would prefer to uset he common keys from `homeassistant.const` like `CONF_IP` and `CONF_PORT`